### PR TITLE
INS-1427: Consensus opencensus

### PR DESCRIPTION
**Pull request by ooki2day**

### CODE REVIEW

File: consensus/phases/communicator.go [hunk 1]
```diff
@@ -142,7 +143,8 @@ func (nc *NaiveCommunicator) sendRequestToNodesWithOrigin(originClaim *packets.N
 
 	for ref, req := range requests {
 		go func(node core.RecordRef, consensusPacket packets.ConsensusPacket) {
-			metrics.ConsensusPacketsSent.WithLabelValues("Request with origin").Inc()
+			stats.Record(context.Background(), consensus.ConsensusPacketsSent.M(1))
+			stats.Record(context.Background())
```
Comment (AndreyBronin) [inline consensus/phases/communicator.go]: ??

File: consensus/phases/secondphase.go [hunk 2]
```diff
@@ -199,7 +205,10 @@ func (sp *SecondPhaseImpl) Execute21(ctx context.Context, pulse *core.Pulse, sta
 		}
 	}
 
-	metrics.ConsensusPacketsRecv.WithLabelValues("phase 21").Add(float64(len(results)))
+	err = stats.RecordWithTags(ctx, []tag.Mutator{tag.Upsert(consensus.TagPhase, "phase 21")}, consensus.ConsensusPacketsRecv.M(int64(len(results))))
+	if err != nil {
+		log.Warn("[ NET Consensus phase-2.1 ] failed to record a metric")
```
Comment (AndreyBronin) [inline consensus/phases/secondphase.go]: logger.Warn

File: consensus/metrics.go [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+package consensus
+
+import (
+	"github.com/insolar/insolar/instrumentation/insmetrics"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+var (
```
Comment (AndreyBronin) [inline consensus/metrics.go]: all exported things should be commented

File: network/nodenetwork/nodekeeper.go [hunk 4]
```diff
@@ -400,7 +401,7 @@ func (nk *nodekeeper) MoveSyncToActive(ctx context.Context) error {
 	inslogger.FromContext(ctx).Infof("[ MoveSyncToActive ] New active list confirmed. Active list size: %d -> %d",
 		len(nk.active), len(mergeResult.ActiveList))
 	nk.active = mergeResult.ActiveList
-	metrics.ConsensusActiveNodes.Set(float64(len(nk.active)))
+	stats.Record(context.Background(), consensus.ConsensusActiveNodes.M(int64(len(nk.active))))
```
[the remark this answers is not in the document]
Comment (ooki2day) [reply network/nodenetwork/nodekeeper.go]: overlooked. fixed

File: consensus/metrics.go (hunk 3 above)
Comment (ooki2day) [reply consensus/metrics.go]: fixed

File: consensus/phases/communicator.go (hunk 1 above)
Comment (ooki2day) [reply consensus/phases/communicator.go]: fixed

**Pull request merged**